### PR TITLE
Add 'echo' command

### PR DESCRIPTION
**Pull request by pknight24**
Add 'echo' command for shellrb. 
Open to any and all feedback.
Tested and runs on macOS, if there is anything else I need to do in the way of testing, please let me know.

### CODE REVIEW

File: lib/packages/echo.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# A simple echo command for shellrb
+class Echo
+  def self.main (params)
```
[the remark this answers is not in the document]
Comment (faraazahmad) [reply lib/packages/echo.rb]: we don't put parenthesis for `params` in shellrb, it looks cleaner.

**Comment by faraazahmad** [comment]
`params[1..(params.length - 1)].join(' ')` is not very easy to understand without comments explaining it. One of the goals with ShellRB is to be as friendly to new contributors as possible.

File: lib/packages/echo.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    if params[0] == '-n'
```
Comment (faraazahmad) [inline lib/packages/echo.rb]: use `case params[0]` instead of `if`

File: lib/packages/echo.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+      print params[1..(params.length - 1)].join(' ')
```
Comment (faraazahmad) [inline lib/packages/echo.rb]: this code is not very "DRY". you could do
`params[start..(params.length - 1)].join(' ')`
and just change value of start according to the conditions

**Comment by pknight24** [comment]
I made some changes as you requested. If there is anything else that I need to do, please let me know.

**Comment by faraazahmad** [comment]
Great!

**Comment by pknight24** [comment]
Thanks! If there are any issues to fix or features to add, I would be happy to help.

**Pull request merged**